cyclonedx/model/component_evidence.py

```diff
@@ -654,6 +672,7 @@ def __init__(
     @property
     @serializable.view(SchemaVersion1Dot5)
     @serializable.view(SchemaVersion1Dot6)
+    @serializable.type_mapping(_IdentitySetSerializationHelper)
```

Comment:
i don't understand why this should be needed. 
current implementation already supports iterable and single value. 
- see https://github.com/OxPirates/cyclonedx-python-lib/blob/8aac64397e93c23b964c83c00d62286c661376d1/cyclonedx/model/component_evidence.py#L638
- see https://github.com/OxPirates/cyclonedx-python-lib/blob/8aac64397e93c23b964c83c00d62286c661376d1/cyclonedx/model/component_evidence.py#L663-L668